src/readline/history.c

```diff
@@ -81,6 +81,9 @@ static char *xmalloc (), *xrealloc ();
 #define member(c, s) ((c) ? index ((s), (c)) : 0)
 #endif
 
+extern close();
```

Comment:
everything in `src/readline` should not be touched as this is an copied library. It should be removed in the near future from the source tree.

Reply:
That would have been ideal. However, we have switched from C98 to C11, so a few things needed update.